Elevel/Application/Features/AuditionCommands/GetAuditionDetailQuery.cs

```diff
@@ -0,0 +1,77 @@
+﻿using AutoMapper;
+using AutoMapper.QueryableExtensions;
+using Elevel.Application.Infrastructure;
+using Elevel.Application.Interfaces;
+using Elevel.Domain.Enums;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Elevel.Application.Features.QuestionCommands;
+
+namespace Elevel.Application.Features.AuditionCommands
+{
+    public class GetAuditionDetailQuery
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid Id { get; set; }
+        }
+
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            private readonly IMapper _mapper;
+
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var audition = await _context.Auditions
+                    .Include(x=>x.Questions).ThenInclude(i => i.Answers)
```

Comment:
please map here with ProjectTo and you can omit Include and ThenInclude

Reply:
Done